Shared/UserAgent.swift

```diff
@@ -85,4 +85,26 @@ public class UserAgent {
 
         return firefoxUA
     }
+
+    public static func desktopUserAgent(defaults: NSUserDefaults) -> String {
+        let userAgent = NSMutableString(string: defaultUserAgent(defaults))
+
+        // Spoof platform section
+        let platformRegex = try! NSRegularExpression(pattern: "\\([^\\)]+\\)", options: [])
+        guard let platformMatch = platformRegex.firstMatchInString(userAgent as String, options:[], range: NSMakeRange(0, userAgent.length)) else {
+            print("Error: Unable to determine platform in UA.")
+            return String(userAgent)
+        }
+        userAgent.replaceCharactersInRange(platformMatch.range, withString: "(Macintosh; Intel Mac OS X 10_11_1)")
+
+        // Strip mobile section
+        let mobileRegex = try! NSRegularExpression(pattern: " FxiOS/[^ ]+ Mobile/[^ ]+", options: [])
```

Comment:
It does feel a little weird to add it only to strip it out again, but IMO not worth any refactoring changes for `defaultUserAgent` (I agree that we'd need to change what we cache).